share/spice/maps/maps/maps_maps.js

```diff
@@ -6,7 +6,7 @@ DDG.require('maps',function(){
         // Mapbox sends back a bunch of places, just want the first one for now
         response = response.features[0];
 
-        if (response.relevance < 0.9) {
+        if (!DDG.isRelevant(response.place_name.toLowerCase()) && response.relevance < 0.9) {
```

Comment:
Is it worth validating that `response.place_name` exists on the response object before calling toLowerCase() on it?